nuxeo-core/nuxeo-core-management-test/src/test/java/org/nuxeo/ecm/core/management/test/probes/TestProbes.java

```diff
@@ -60,6 +62,20 @@
     @Inject
     ProbeManager pm;
 
+    @Inject
+    FakeService fs;
+
+    protected static final int TEST_INTERVAL_SECONDS = -1;
+
+    @Before
+    public void setupCacheOnProbes() {
+        // remove effects linked to cache for these tests
+        Framework.getProperties().setProperty(ProbeManagerImpl.DEFAULT_HEALTH_CHECK_INTERVAL_SECONDS_PROPERTY,
+                String.valueOf(TEST_INTERVAL_SECONDS));
```

Comment:
We usually remove the property completely instead of setting a default value (the user of the property should have a default anyway).

Reply:
done 👍